source/renderer/app/components/widgets/forms/ItemsDropdown.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import React, { Component } from 'react';
+import { filter, escapeRegExp } from 'lodash';
+import { Select } from 'react-polymorph/lib/components/Select';
+import { SelectSkin } from 'react-polymorph/lib/skins/simple/SelectSkin';
+import ItemDropdownOption from './ItemDropdownOption';
+import type { ItemDropdown } from './ItemDropdownOption';
+
+/**
+ *
+ * This component extends the React Polymorph's Select component
+ * Any prop from it can be used
+ * Reference:
+ * https://github.com/input-output-hk/react-polymorph/blob/develop/source/components/Select.js
+ *
+ * For a simple usage, it only requires an `option` array, with an extra `detail` item:
+ *
+ * [
+ *   {
+ *     label: 'Top content',
+ *     detail: 'Bottom content',
+ *     value: '...'
+ *   }
+ * ]
+ *
+ */
+export type ItemDropdownProps = {
+  options: Array<ItemDropdown>,
+};
+
+export const onSearchItemsDropdown = (
```

Comment:
Same here … something like `filterOptionsBySearchValue`

Reply:
I'd keep it for the [same reason](https://github.com/input-output-hk/daedalus/pull/2540#discussion_r626009138)